src/ontology/util/OntologyStatisticUtil.java

```diff
@@ -150,7 +150,7 @@ private static String getHashCode(OntologyValue... values) {
      * @param constraints
      * @param qualifierHierarchy
      */
-    public static void verifySolution(InferenceSolution mergedSolution,
+    public static void verifySolution(InferenceResult mergedSolution,
```

Comment:
`mergedSolution` -> `mergedResult`

Reply:
Is `verifySolution()` only called if there is solution? If yes, adding `assert mergedResult.hasSolution()` would be good. If `verifySolution()` is accidentally called if no solution, calling `mergedResult.getSolutionForVariable()` throws NPE.

Reply:
yes, `verifySolution` should be called if there is a solution. Let me add the assertion.